specs/examples/README.md

```diff
@@ -0,0 +1,157 @@
+  - While additional routes in examples are not prevented, they should be discouraged except for the use-case of demonstrating routing. It's likely extra routes make for a less coherent and targeted example anyway.
+
+### Migrate CDN Builds to a separate package
+
+Part of this proposal is to migrate the prebuilt all-components bundles from nimble-components to a separate npm package.
+
+This has a couple benefits:
+- Avoids confusion within the Nimble Components package. Currently nothing prevents a user from accidentally mixing usage of the normal nimble component es modules, the all-components.js es module, or the all-components-bundle.js built output. 
+
+  Mixing them is very likely to break the application in hard to predict ways by creating multiple singletons in global scope, duplicating element registrations, etc. Moving them all to a separate package can eliminate possible confusion.
+- Can inline all the related content, i.e. the needed nimble fonts can be copied from nimble-tokens into the cdn package making the package standalone. Doing that with the nimble-components package would not be appropriate.
+- Proposed package name `@ni/nimble-bundle`
+    - Alternative names:
+        - `@ni/nimble-bundled`
+        - `@ni/nimble-components-cdn`
+        - `@ni/nimbl-components-bundle`
+        - `@ni/nimble-cdn-bundle`
+        - `@ni/nimble-cdn`
+- Part of this proposal would also be a breaking change to the Blazor package structure. We can leverage the `@ni/nimble-bundle` package directly in build instead of building a custom `wwwroot`.
+  - That would change from:
+    ```
+    - wwwroot/
+      - nimble-components/all-components-bundle.min.js
+      - nimble-tokens/css/fonts.css
+      - nimble-tokens/assets/fonts.css
+    ```
+    to the following where dist is a direct copy of the nimble-bundle `dist` output:
+    ```
+    - wwwroot/
+      - dist/all-components-bundle.min.js
+      - dist/css/fonts.css
+      - dist/assets/fonts.css
+    ```
+  - Alternatives:
+    - Don't introduce a breaking change, keep the existing `wwwroot` file structure in Nimble Blazor
+      - Pros: Not a breaking change
+      - Cons: Nimble Blazor continues to maintain an alternate structure of bundled output
+
+### File structure
+
+Examples will be organized as follows:
+
+```
+- examples/
+    - README.md describing folder layout and documenting example expectations (linting, etc).
+    - <framework>/
+        - README.md: Table of contents for the examples
+        - getting-started/
+            - README.md: Should include a Stackblitz link to open the example immediately in addition to other content
+            - package.json
+            - source/
+        - <other examples>: TBD
+```
+
+Notes:
+- A subfolder per `<framework>`, i.e.: `vanilla`, `cdn`, `angular`, `blazor`
+- Examples should be specific and targeted
```

Comment:
Will each example be totally standalone or would we have an Angular workspace containing an application per example?